tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -127,11 +127,11 @@ EpochSyncProofLastEpochData = 1316817317
 EpochSyncProofV1 = 2731130859
 EpochValidatorInfo = 1707010927
 ExecutionMetadata = 2384490761
-ExecutionOutcome = 1141507036
-ExecutionOutcomeWithId = 1104599764
-ExecutionOutcomeWithIdAndProof = 28425232
-ExecutionOutcomeWithProof = 1694747862
-ExecutionStatus = 1514099984
+ExecutionOutcome = 2674331133
+ExecutionOutcomeWithId = 553081407
+ExecutionOutcomeWithIdAndProof = 537041540
+ExecutionOutcomeWithProof = 3950405827
+ExecutionStatus = 268265901
```

Comment:
Are these merge conflict artifacts? This PR should not introduce any protocol changes. Is this due to `MissingTrieValue`? :-(

Reply:
Yes, due to MissingTrieValue. Should we switch it back? It causes problems with the spec

Reply:
@saketh-are hey, could you comment on this? Do I need to do something to support new versions of these structures?

Would also love to hear your opinion about https://github.com/near/nearcore/pull/12961#discussion_r2064483422

Reply:
@saketh-are Which serialization format does it use? For borsh a tuple of two fields is fully compatible with a struct of two named fields.

The reason we wanted to change is to improve the OpenAPI spec and the JSON RPC output since currently `InvalidTxError` can return `StorageError` with `MissingTrieValue` variant with an array `["MissingTrieValueContext enum variant", "base58-hash"]`, e.g. `["trie_iterator", "6Kea5WQ151QMBTurjuJotKvfrtx4Fe2FKjMT4vUJSyoW"]`. It is tricky to make a spec for such a structure and as such I suggested to turn it into `{"context": "trie_iterator", "hash": "6Kea5WQ151QMBTurjuJotKvfrtx4Fe2FKjMT4vUJSyoW"}`, which is easier to consume for JSON RPC users and easier to document with OpenAPI spec.